exercises/relative-distance/instructions.md

```diff
@@ -0,0 +1,34 @@
+# Instructions
+
+Your task is to determine the degree of separation between two individuals in a family tree.
+
+- You will be given an input, with all parent names and their children.
+- Each name is unique, a child _can_ have one or more parents.
+- The degree of separation is defined as the shortest number of connections from one person to another.
+- If two individuals are not connected, return a value that represents "no relation."
+  Please see the test cases for the actual implementation.
+
+## Example
+
+Given the following family tree:
+
+```text
+      ┌──────────┐            ┌──────────┐ ┌───────────┐
+      │  Helena  │            │  Erdős   │ │  Shusaku  │
+      └───┬───┬──┘            └─────┬────┘ └──────┬────┘
+      ┌───┘   └───────┐             └──────┬──────┘
+      ▼               ▼                    ▼
+┌──────────┐     ┌────────┐          ┌──────────┐
+│   Isla   │     │  Tariq │          │   Kevin  │
+└────┬─────┘     └────┬───┘          └──────────┘
```

Comment:
Nice touch, that must be Kevin Bacon...

Reply:
It is. Thanks to @BNAndras for the trip down memory lane when we did this with other film nerds in my "sneak preview" circle.